test/functional/uvm_mem_backingtype_test.go

```diff
@@ -104,7 +102,6 @@ func BenchmarkMemBackingTypeVirtualLCOW(b *testing.B) {
 
 	require.Build(b, osversion.RS5)
 	requireFeatures(b, featureLCOW)
-	logrus.SetOutput(io.Discard)
```

Comment:
>logrus.SetOutput(io.Discard) [](http://example.com/codeflow?start=1&length=28)

Are the changes in this file related to the exclude or any feature changes in some way?

Reply:
nope, it looks like they snuck in from another branch